package.json

```diff
@@ -82,5 +82,8 @@
   "workspaces": [
     "packages/*"
   ],
-  "packageManager": "yarn@4.2.2"
+  "packageManager": "yarn@4.2.2",
+  "resolutions": {
+    "@sendbird/chat": "file:./sendbird-chat-4.14.2.tgz"
```

Comment:
Is there PR for this change?

Reply:
Not yet, after being deployed to specific regions, Chat SDK will be released once it has been deployed to all regions.

Branch: https://github.com/sendbird/chat-js/tree/feature/danney/v4/CLNP-4946-Add-message-property-in-FileMessage-240827